Tests/Realm.Tests/Sync/SyncTestBase.cs

```diff
@@ -197,10 +197,10 @@ protected async Task<FlexibleSyncConfiguration> GetFLXIntegrationConfigAsync(App
             return UpdateConfig(new FlexibleSyncConfiguration(user, optionalPath));
         }
 
-        protected async Task<Realm> GetFLXIntegrationRealmAsync(App app = null)
+        protected async Task<Realm> GetFLXIntegrationRealmAsync(App app = null, FlexibleSyncConfiguration flxConfig = null)
         {
-            var config = await GetFLXIntegrationConfigAsync(app);
-            return await GetRealmAsync(config);
+            flxConfig ??= await GetFLXIntegrationConfigAsync(app);
+            return await GetRealmAsync(flxConfig);
```

Comment:
Why this change? If you pass in the config, it's better to just use `GetRealmAsync` directly.

Reply:
I don't understand this comment. If the config is passed, it already directly uses `GetRealmAsync`.